perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/json/registrarManager/GetFormItems.java

```diff
@@ -261,21 +270,43 @@ public void onFinished(JavaScriptObject jso){
 			type_label.setTitle(item.getType());
 			ft.setWidget(i, 1, type_label);
 
-			// 2 = preview
+			// 2 = disabled
+			if(item.getDisabled() != "NEVER") {
+				Image lockIcon =  new Image(SmallIcons.INSTANCE.lockIcon());
+				String dependency = item.getDisabledDependencyItemId() == 0 ? "" : items.stream()
```

Comment:
This might fix the label when the dependency is not selected.
```suggestion
				String dependency = item.getDisabledDependencyItemId() == 0 ? "it" : items.stream()
```